wp-content/themes/core/theme.json

```diff
@@ -95,10 +95,100 @@
         }
       ]
     },
+    "custom": {
+      "fontWeight": {
```

Comment:
I added these as custom WP presets because I've seen different typefaces have differing numeric values for font-weight names. For example, one font will define bold 800 and different font will define it as 700. 

I could be overcomplicating it, but variables help me not have to remember the specific project's font-weight values.